lib/js/components/SelectList/SelectListItemTile/SelectListItemTile.stories.ts

```diff
@@ -0,0 +1,57 @@
+import SelectListItemTile from './SelectListItemTile.vue';
+import { TILE_STATES } from './../../Tile/Tile.consts';
+
+import { Args, Meta, StoryFn } from '@storybook/vue3';
+import { args, argTypes, data, template } from './../../Tile/Tile.sb.shared';
+
+export default {
+	title: 'Components/SelectListItemTile',
+	component: SelectListItemTile,
+} as Meta<typeof SelectListItemTile>;
+
+const StoryTemplate: StoryFn<typeof SelectListItemTile> = (args) => ({
+	components: { SelectListItemTile },
+	setup() {
+		return { ...args };
+	},
+	template: template('select-list-item-tile'),
+	data,
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+Interactive.argTypes = argTypes;
+Interactive.args = args;
+
+Interactive.parameters = {
+	actions: {
+		handles: ['click'],
+	},
+	design: {
+		type: 'figma',
+		url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=1923%3A34378',
+	},
+};
+
+const StoryStaticTemplate: StoryFn<typeof SelectListItemTile> = (args) => ({
+	components: { SelectListItemTile },
+	setup() {
+		return { ...args };
+	},
+	template: template('select-list-item-tile'),
+	data,
+});
+
+export const Static = StoryStaticTemplate.bind({});
+
+Static.argTypes = argTypes;
```

Comment:
why we use controls in Static template?

Reply:
hmm. I think I copied it without thinking :D